12-spatial-cv.Rmd

```diff
@@ -274,7 +274,7 @@ It acts as a 'meta-package', providing a unified interface to popular supervised
 The standardized **mlr3** interface is based on eight 'building blocks'.
 As illustrated in Figure \@ref(fig:building-blocks), these have a clear order.
 
-(ref:building-blocks) Basic building blocks of the mlr3 package. Source: @bischl_applied_2024. (Permission to reuse this figure was kindly granted.)
+(ref:building-blocks) Basic building blocks of the mlr3 package. @bischl_applied_2024. Permission to reuse this figure was kindly granted.
```

Comment:
```suggestion
(ref:building-blocks) Basic building blocks of the mlr3 package @bischl_applied_2024. Permission to reuse this figure was kindly granted.
```

Reply:
yeah, I was wondering about this as well, thanks!